### ISSUE TITLE
Fusion installer script: Wrong name for Nested ESXi

### ISSUE THREAD

**Issue by pdellaert** [vmware/photon-controller#21]
[The script](https://bintray.com/artifact/download/photon-controller/public/UserWorkflow/PhotonControllerInstallMac.sh) to automatically setup a [Fusion environment](https://github.com/vmware/photon-controller/wiki/fusion-installation) has an old name for the nested ESXi appliance.

Line 19:
```
ESX_VM_NAME="nested_esxi_appliance_v6.0"
```

Should be:
```
ESX_VM_NAME="nested_esxi_appliance_v6.0u2"
```

**Comment by mreferre** [comment]
Hi Philippe. 

I am the (co-)author of the script. Good catch. FYI that script was built for the previous version of the code (that had a slightly more rigid install procedure). In fact the install was (basically) an export of a live environment as a (Photon Controller) appliance that could drive an ESXi host (this could be 6.0 or 6.0 U2) 

With the new 0.8 release (among the many new features) we have released a different (much better) way to install and test Photon Controller (more dynamic, more automated etc). 

Long story short, the script we link is still working BUT will get you the previous bits. Yes we could have updated the ESXi image link but that would not made a huge difference. 

We are working on a new script that will leverage the new setup workflows (and hopefully will provide more flexibility than the current script). 

HTH.

**Comment by pdellaert** [comment]
Ah, Great :)! I just saw the wiki page on running it and decided to try it. Of course, any better and more improved way, is always nice.
